bundle/edu.gemini.p2checker/src/main/java/edu/gemini/p2checker/rules/nifs/NifsRule.java

```diff
@@ -184,17 +185,17 @@ public IConfigMatcher getMatcher() {
         P2Problems prob = new P2Problems();
         public IP2Problems check(ObservationElements elements)  {
             for (ObsContext ctx : elements.getObsContext()) {
-                final SPTarget baseTarget       = ctx.getTargets().getBase();
+                final Asterism asterism       = ctx.getTargets().getAsterism();
                 final Option<SPTarget> oiTarget = getOITarget(ctx);
                 final Option<SPTarget> aoTarget = getAOTarget(ctx);
 
-                if (baseTarget == null || oiTarget.isEmpty() || aoTarget.isEmpty()) return null;
+                if (asterism == null || oiTarget.isEmpty() || aoTarget.isEmpty()) return null;
```

Comment:
There was always supposed to be a base position, and there should similarly always be an asterism now.  I guess it's not worth going through and removing the `null` checks though?